config/brakeman.yml

```diff
@@ -0,0 +1,10 @@
+---
+:min_confidence: 3
+:ignore_ifs: false
+:skip_libs: false
+:exit_on_warn: false
```

Comment:
Since there are currently warnings, I have set `exit_on_warn` to false so that the build can pass, and I think fixing the warnings is outside the scope of this PR.